new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleNginxValidation.java

```diff
@@ -628,4 +673,149 @@ private void checkServiceCreated(String serviceName) {
+        .withFailMessage(String.format("can not scale cluster %s of domain %s in namespace %s",
+            clusterName, domainUid, domainNamespace))
+        .isTrue();
+
+    // generate a curl command to ping the sample app through the ingress controller
+    curlCmd = generateCurlCmd(clusterName);
+
+    // generate the expected server list which should be in the sample app response string
+    List<String> expectedServerNamesInAppResponse =
+        generateExpectedServerNamesInAppResponse(clusterName, replicasBeforeScale);
+    logger.info("expected server name list which should be in the sample app response: {0} before scale",
+        expectedServerNamesInAppResponse);
+
+    if (replicasBeforeScale <= numberOfServers) {
+
+      // check that NGINX can access the sample apps from the original managed servers in the domain
+      logger.info("Check that NGINX can access the sample app from the original managed servers in the domain "
+                  + "while the domain is scaling up.");
+      assertThat(callWebAppAndCheckForServerNameInResponse(curlCmd, expectedServerNamesInAppResponse, 50))
+          .as("Verify NGINX can access the sample app from the original managed servers in the domain")
+          .withFailMessage("NGINX can not access the sample app from one or more of the managed servers")
+          .isTrue();
+
+      // check new managed server pods are created and wait for them to be ready
+      for (int i = replicasBeforeScale + 1; i <= numberOfServers; i++) {
+        String manageServerPodName = manageServerPodNamePrefix + i;
+
+        // check new managed server pods are created
+        logger.info("Check for the new managed server pod {0} is created in namespace {1}",
+            manageServerPodName, domainNamespace);
+        checkPodCreated(manageServerPodName);
+
+        // check new managed server pods are ready
+        logger.info("Wait for the new managed server pod {0} to be ready in namespace {1}",
+            manageServerPodName, domainNamespace);
+        checkPodReady(manageServerPodName);
+
+        // check new managed server services are created
+        logger.info("Check for the new managed server service {0} is created in namespace {1}",
+            manageServerPodName, domainNamespace);
+        checkServiceCreated(manageServerPodName);
+
+        // add the new managed server to the list
+        expectedServerNamesInAppResponse.add(clusterName + "-managed-server" + i);
+
+      }
+
+      // check that NGINX can access the sample apps from new and original managed servers
+      logger.info("Check that NGINX can access the sample app from the new and original managed servers "
+          + "in the domain after the domain is scaled up.");
+      assertThat(callWebAppAndCheckForServerNameInResponse(curlCmd, expectedServerNamesInAppResponse, 50))
+          .as("Check NGINX can access the sample app from all managed servers in the domain")
```

Comment:
this need to be fixed as well.

Reply:
.as() message is not for a successful assertion.